server/index.js

```diff
@@ -1,6 +1,26 @@
+require('dotenv').config({
+  path: '../frontend/.env'
+})
+
 const express = require('express')
 const bodyParser = require('body-parser')
 const Web3 = require('web3')
+const plaid = require('plaid')
+const pino = require('pino')
+
+const logger = pino()
+
+const PLAID_CLIENT_ID = process.env.REACT_APP_PLAID_CLIENT_ID
+const PLAID_SECRET = process.env.REACT_APP_PLAID_SECRET
```

Comment:
We shouldn't do this. This means the secret will be in the bundle.js, which is a problem.